main.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <iomanip>
+
+// Stock Program
+// By Clayton Checkeye for CPT_180
+
+int main() {
+        // Declarations
+
+    const int SHARES_BOUGHT = 1000;
+    const int SHARES_SOLD = 1000;
+    const double COMMISSION_RATE = 0.2;
+    const double PURCHASE_PRICE = 45.50;
+    const double SALE_PRICE = 56.90;
+
+    //Calculations
+
+    double amount_paid = SHARES_BOUGHT * PURCHASE_PRICE;
+    double initial_commission = COMMISSION_RATE * amount_paid;
```

Comment:
I would recommend finding a better name for `inital_commission` and `second_commission`. Consider what each variable is associate with as far as a larger process of the program.